### ISSUE TITLE
Can't repeat your results on TensorFlow 0.12.1   

### ISSUE THREAD

**Issue by mdsalnikov** [Zeta36/tensorflow-image-wavenet#2]
My best loss = 0.035 on 25k steps. 

@Zeta36 , can you give me your trained model?
